ai-docs/emoji-commit-ref.yaml

```
@@ -190,3 +190,85 @@
+      - 'references'
+      - 'part of'
+      - 'related to'
+      - 'contributes to'
+      - 'towards'
+  
+  format_examples:
+    github_issues:
+      - pattern: '#<issue-number>'
+        example: 'Closes #42'
+        description: 'References issue in the current repository'
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)